### PR TITLE
Feature/chan persistance

### PR DESCRIPTION
**Pull request by digination**
No additional comments were issued on #714 , so i guess my PR is ok for merging :)

C.

### CODE REVIEW

File: app/controllers/account.js [hunk 1]
```diff
@@ -84,12 +84,14 @@ module.exports = function() {
...
         },
+
         profile: function(req, res) {
             var form = req.body || req.data,
                 data = {
                     displayName: form.displayName || form['display-name'],
                     firstName: form.firstName || form['first-name'],
-                    lastName: form.lastName || form['last-name']
+                    lastName: form.lastName || form['last-name'],
+                    rooms: form.rooms,
```
Comment (hhaidar) [inline app/controllers/account.js]: do you mind referring to `rooms` as `openRooms` or something?

File: media/js/client.js [hunk 2]
```diff
@@ -275,8 +259,18 @@
...
-        RoomStore.remove(id);
+        // Remove room id from User room list.
+
+        var urooms = this.user.get('rooms');
+        var urindex =  urooms.indexOf(id);
+        if (urindex != -1) {
+          urooms.splice(urindex, 1);    
+        }
+
+        console.log(urooms);
```
Comment (hhaidar) [inline media/js/client.js]: extra console

File: app/models/user.js [hunk 3]
```diff
@@ -87,8 +87,8 @@ var UserSchema = new mongoose.Schema({
         trim: true
     },
     rooms: [{
-		type: ObjectId,
-		ref: 'Room'
```
Comment (hhaidar) [inline app/models/user.js]: yea I'd much prefer if we used a new property dedicated to this

**Comment by hhaidar** [comment]
@digination thank you for doing this!

**Comment by digination** [comment]
Ok, will make a few mods and resend (probable tomorrow)
Cheers,

C.

**Comment by digination** [comment]
Ok i made some improvements, it should be ok now :)

**Comment by digination** [comment]
@hhaidar Hi, can i get an update on the last commit, Is it ok for you for mergin ? 
Cheers,
Clément.

**Comment by hhaidar** [comment]
thanks again @digination :D

**Pull request merged**